src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -302,10 +302,16 @@ const AccountClosure = (() => {
             collapsible: true,
             active     : true,
         });
+        $('#back').on('click', () => $('#account-closure-dialog').css('display', 'none'));
```

Comment:
let's store `$('#account-closure-dialog')` in a global variable so we don't have to keep fetching it every time on clicking back, deactivate, and form submission

Reply:
can we use `.setVisibility(0)` instead of setting css display to none?